requirement.txt

```diff
@@ -0,0 +1,48 @@
+appnope==0.1.3
```

Comment:
any reason why this is added at this stage?

ideally all of these has to be already installed right?

Reply:
for next time not have to install dependencies one by one so added requirement file